com.github.Qv2ray.json

```diff
@@ -0,0 +1,85 @@
+{
+    "app-id": "com.github.Qv2ray",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "qv2ray",
+    "rename-desktop-file": "qv2ray.desktop",
+    "rename-icon": "qv2ray",
+    "rename-appdata-file": "qv2ray.metainfo.xml",
```

Comment:
While you're at it, you can just rename those files upstream. The desktop and metainfo should be named following the app-id; regarding the icon, you don't have to rename it upstream but Flatpak only exports the files that are prefixed with the app-id, so renaming it makes sense.

Reply:
> While you're at it, you can just rename those files upstream. The desktop and metainfo should be named following the app-id; regarding the icon, you don't have to rename it upstream but Flatpak only exports the files that are prefixed with the app-id, so renaming it makes sense.

It **IS** able to change the desktop filename, however, doing so will break the build of Ubuntu PPA and ArchAUR, we'll contact those packagers to discuss whether it is necessary to change the filename or just use the `rename-*` settings, as shown above.

Reply:
Well, I don't see how those files might break the build of a deb/aur package. Anyway, that's not a blocker, but a suggestion.